clrdefinitions.cmake

```diff
@@ -15,6 +15,7 @@ if (CLR_CMAKE_TARGET_ARCH_AMD64)
   if (CLR_CMAKE_PLATFORM_UNIX)
     add_definitions(-DDBG_TARGET_AMD64_UNIX)
   endif()
+  add_definitions(-DBIT64=1)
   add_definitions(-D_TARGET_AMD64_=1)
```

Comment:
Currently, ``BIT32`` and ``BIT64`` are defined only in PAL. I revised ``clrdefinitions.cmake`` to defined ``BIT32`` / ``BIT64`` appropriately, and use the to revise ``stackwalk.cpp``